rllib/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -373,49 +360,6 @@ def training_step(self) -> ResultDict:
         # Return all collected metrics for the iteration.
         return train_results
 
-    @staticmethod
-    @override(Algorithm)
-    def execution_plan(
-        workers: WorkerSet, config: AlgorithmConfigDict, **kwargs
-    ) -> LocalIterator[dict]:
-        assert (
-            len(kwargs) == 0
-        ), "Alpha zero execution_plan does NOT take any additional parameters"
```

Comment:
I've merged your PR into this PR because it removes lots of code from IMPALA and I wanted to avoid dealing with all that code. As soon as the execution plan removal PR is merged, I can merge master into this PR again and you won't see these changes when I ask for review.